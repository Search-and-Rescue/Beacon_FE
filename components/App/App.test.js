import React from 'react';

describe('App', () => {

  it('should pass this test', () => {
    expect(true).toEqual(true);
  });

});