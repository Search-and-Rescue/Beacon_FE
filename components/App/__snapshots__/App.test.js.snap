// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot with all of the data passed through 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Component />
  ;
</Provider>
`;
